x-pack/plugins/secops/public/components/timeline/data_providers/icons_footer.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiIcon, EuiToolTip } from '@elastic/eui';
+import { isEmpty } from 'lodash/fp';
+import moment from 'moment';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+import { DataProvider } from './data_provider';
+
+const HorizontalBar = styled(EuiHorizontalRule)`
+  margin: 4px 0px;
+`;
+
+const GroupIcons = styled(EuiFlexGroup)`
+  width: 100%;
+`;
+
+interface OwnProps {
+  dataProvider: DataProvider;
+}
+export const IconsFooter = pure<OwnProps>(({ dataProvider }: OwnProps) => {
+  if (!dataProvider.queryDate || (dataProvider.queryDate && isEmpty(dataProvider.queryDate))) {
+    return null;
+  }
+  const dates = dataProvider.queryDate.trim().match(/\d+/g);
+  const tooltipStr = `${moment(parseInt(dates![0], 10)).format('L LTS')} - ${moment(
+    parseInt(dates![1], 10)
+  ).format('L LTS')}`;
```

Comment:
I wonder if we're going to have to write a ticket to use the built in date time formatters eventually from advanced:

<img width="919" alt="screen shot 2019-01-09 at 4 37 31 pm" src="https://user-images.githubusercontent.com/1151048/50936437-eb482080-142c-11e9-9be4-069c96371f08.png">

We could use it on the backlog. Just an observation looking at our choices of formats for date times.

Added a ticket for it: https://github.com/elastic/ingest-dev/issues/199

Reply:
Oh yes that's for sure

Reply:
I did even know this one